Resources/config/yui.yml

```diff
@@ -297,6 +301,23 @@ system:
                 dashboardview-ez-template:
                     type: 'template'
                     path: %ez_platformui.public_dir%/templates/dashboard.hbt
+                ez-dashboardblockbasedview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'dashboardblockbasedview-ez-template'
+                    path: %ez_platformui.public_dir%/js/views/ez-dashboardblockbasedview.js
+                dashboardblockbasedview-ez-template:
+                    type: 'template'
+                    path: %ez_platformui.public_dir%/templates/dashboard/dashboard.hbt
```

Comment:
to be consistent with the path of the view, this template should not be in the dashboard directory (but indeed this directory will be useful the block templates).